apps/web/src/pages/block/[id].tsx

```diff
@@ -47,6 +49,7 @@ const Block: NextPage = function () {
 
     return deserializeFullBlock(rawBlockData);
   }, [rawBlockData]);
+  const { data: latestBlock } = api.block.getLatestBlock.useQuery();
```

Comment:
We're already fetching this data in the `ExplorerDetails` component. 

If we need to access it in other parts of the component tree, it might be better to create a context now so we can consume it wherever necessary. What do you think?
wdyt?

Reply:
We no longer use `latestBlock` after commit [83dcc29](https://github.com/Blobscan/blobscan/pull/479/commits/83dcc2943977815edb28f3e55070c47de385decf).